src/api/changePassword.js

```diff
@@ -0,0 +1,48 @@
+const router = require("express").Router();
+const nodemailer = require("nodemailer");
+const cors = require("cors");
+const User = require("../models/User");
+const bodyParser = require("body-parser");
+const jwt = require("jsonwebtoken");
+const exjwt = require("express-jwt");
+const sendResponse = require("../utils/sendResponse");
+const { SECRET_TOKEN } = require("../utils/secret-token");
+
+router.post("/changePassword", async function(req, res) {
+  if (!req.body || !req.body.email || !req.body.token || !req.body.password) {
```

Comment:
As long as it has the token, old password, and new password, it has enough information to change the password. You can get the email by looking the user up in the database.

Reply:
Yep, that makes sense!